backend/src/api/interview.js

```diff
@@ -1,15 +1,26 @@
 const express = require('express')
+const keyPath =
+  process.env.NODE_ENV === 'test' ? '../../tests/artifacts/test-keys.json' : process.env.KEY_JSON
+const keyData = require(keyPath)
 const router = express.Router()
 const { errorWrap } = require('../middleware')
 const { Interview } = require('../models')
 var mongodb = require('mongodb')
 
 router.get(
-  '/verified-interviewer',
+  '/verify_interviewer/:key',
   errorWrap(async (req, res) => {
-    // const foundKey = await Interview.findById(req.params.interviewerKey);
-    // res.json({ result: foundKey })
-    /*** NOT IMPLEMENTED */
+    let keyVerified = false
+    if (req.params.key && req.params.key.length === 11) {
+      keyVerified = keyData.keys.filter(currKey => currKey.key === req.params.key).length !== 0
+    }
+    let statusCode = keyVerified ? 200 : 400
+    let message = keyVerified ? 'key is verified' : 'key did not pass verification'
+    res.json({
+      code: statusCode,
```

Comment:
the http response here has still has a status code of 200 even though ‘statusCode = 400’.  look at how we did it here https://github.com/hack4impact-uiuc/h4i-recruitment/blob/master/backend/src/middleware/errorHandler.js